tensorbay/cli/commit.py

```diff
@@ -32,13 +32,11 @@ def _implement_commit(obj: Dict[str, str], tbrn: str, message: Tuple[str, ...])
         error(f'To commit, "{info}" must be in draft status, like "{info}#1"')
 
     dataset_client.checkout(draft_number=info.draft_number)
-    if message:
-        title, description = message[0], "\n".join(message[1:])
-    else:
-        title, description = edit_input(_COMMIT_HINT.format(dataset_client.get_draft().title))
 
-    if not title:
-        error("Aborting commit due to empty commit message")
+    error_message = "Aborting commit due to empty commit message"
+    draft = dataset_client.get_draft()
+    hint_message = (_COMMIT_HINT, draft.title, draft.description)
```

Comment:
why input a tuple to `edit_draft_info`, can we directly input `HINT`, `title` and `desctiption`?